packages/zowe-explorer-api/src/tree/IZoweTreeNode.ts

```diff
@@ -128,14 +131,20 @@ export interface IZoweDatasetTreeNode extends IZoweTreeNode {
      */
     filter?: Sorting.DatasetFilter;
     /**
+     * @deprecated Please use `getEncodingInMap` and `updateEncodingInMap` instead.
+     *
```

Comment:
general question:
Do we plan to remove all these deprecated methods before V3, or as part of a future V4?

Reply:
This will be for v4 to give extenders enough time to transition 😋